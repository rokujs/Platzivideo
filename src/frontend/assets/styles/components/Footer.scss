.footer {
    background-color: #fff0;
    display: flex;
    align-items: center;
    width: 100%;
    height: 100px;
  }
  
  .footer a {
    padding: 0 15px;
  }