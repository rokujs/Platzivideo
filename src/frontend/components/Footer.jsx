import React from 'react';
import '../assets/styles/components/Footer.scss';

const Footer = () => (
  <footer className='letters footer'>
    <a href='/inicio_de_sesion.html'>Términos de uso.</a>
    <a href='/inicio_de_sesion.html'>Declaración de privacidad.</a>
    <a href='/inicio_de_sesion.html'>Centro de ayuda.</a>
  </footer>
);

export default Footer;
