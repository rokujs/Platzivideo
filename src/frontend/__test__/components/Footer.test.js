import React from 'react';
import { mount } from 'enzyme';
import {create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
    const footer = mount(<Footer />);

    test('Render footer component', () => {
        expect(footer.length).toEqual(1);
    });

    test('Footer have 3 anchors', () =>{
        expect(footer.find('a')).toHaveLength(3);
    });

    test('Footer snapshot', () => {
        const footer = create(<Footer/>);
        expect(footer.toJSON()).toMatchSnapshot();
    })
});