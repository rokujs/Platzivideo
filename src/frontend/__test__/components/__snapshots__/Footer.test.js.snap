// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer snapshot 1`] = `
<footer
  className="letters footer"
>
  <a
    href="/"
  >
    Términos de uso.
  </a>
  <a
    href="/"
  >
    Declaración de privacidad.
  </a>
  <a
    href="/"
  >
    Centro de ayuda.
  </a>
</footer>
`;
